config.py

```diff
@@ -1,7 +1,9 @@
+"""Flask configuration."""
 import os
 
 ENVIRONMENT = os.environ.get("ENVIRONMENT", "development")
 
+# Development env specific
```

Comment:
Let's remove this as we are getting a gist from the next line?

Reply:
Sure